src/app/hooks/keycloak.ts

```diff
@@ -13,18 +13,19 @@ const init = async (config: Config, getInsightsAccessToken: () => Promise<string
   return await initKeycloak(keycloakConfig, getInsightsAccessToken);
 }
 
-export const useAuth = (): Auth => {
+export const useAuth = (): Auth | undefined => {
   const keycloakInstance = useRef<KeycloakInstance>();
   const config = useConfig();
   const insights = useInsights();
 
-  if (config === undefined || insights.chrome.auth === undefined) {
+
+  if (insights.chrome.auth === undefined) {
     throw new Error("useAuth must be used inside a config provider, and insights auth");
```

Comment:
If you need to change this, you need to change the error message too, though I think the most recent change we did makes this unneeded.